code/web/package.json

```diff
@@ -5,6 +5,7 @@
   "main": "index.js",
   "scripts": {
     "setup": "npm install",
+    "test": "jest",
```

Comment:
Fun challenge having to install the testing library & get this script up & running!

Reply:
We just learned about Jest yesterday! We may need some advice in implementing

Reply:
upvote to collaborating on this!